### PR TITLE
Add de.haeckerfelix.Remotely

### PR DESCRIPTION
**Pull request by haecker-felix**
https://gitlab.gnome.org/World/Remotely

### CODE REVIEW

File: de.haeckerfelix.Remotely.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
```
Comment (bilelmoussaoui) [inline de.haeckerfelix.Remotely.json]: Those build options are now used by default on Freedesktop sdk, you can remove them

File: de.haeckerfelix.Remotely.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    "url" : "https://download.gnome.org/sources/gtk-vnc/0.7/gtk-vnc-0.7.1.tar.xz",
+                    "sha256" : "f34baa696615ef67666e8465b4d0ac563355e999a77d2cc42ad4625a24f7aab1"
+                }
+            ]
+        },
+        {
+            "name" : "remotely",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--libdir=lib"
```
Comment (bilelmoussaoui) [inline de.haeckerfelix.Remotely.json]: It's already set by flatpak-builder.

File: de.haeckerfelix.Remotely.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            ],
+            "builddir" : true,
```
Comment (bilelmoussaoui) [inline de.haeckerfelix.Remotely.json]:
```suggestion
            "builddir" : true,
```
Same as the two other comments.

**Comment by bilelmoussaoui** [comment]
bot, build de.haeckerfelix.Remotely

**Comment by haecker-felix** [comment]
bot, build de.haeckerfelix.Remotely

**Comment by haecker-felix** [comment]
bot, build de.haeckerfelix.Remotely

**Comment by TingPing** [comment]
I've probably asked this before but do you not control haeckerfelix.de?

**Comment by haecker-felix** [comment]
I own haeckerfelix.de, but I don't use it for a website. Do you need a verification?

**Comment by TingPing** [comment]
> I own haeckerfelix.de, but I don't use it for a website. Do you need a verification?

I don't think we've ever been overly strict about it but its probably a good policy to have. If you can just make a `txt` record containing `github=haecker-felix` or something that would be nice.

**Comment by haecker-felix** [comment]
I've added a TXT record, you should be able to see it now.

**Comment by nedrichards** [comment]
Are there any other upstream contributors who might need access?

**Comment by haecker-felix** [comment]
@nedrichards no, I don't think so.

**Comment by nedrichards** [comment]
bot, build de.haeckerfelix.Remotely

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/de.haeckerfelix.Remotely

**Pull request closed without merging**